docs/building-rpm-ostree-and-autosd.md

```diff
@@ -0,0 +1,171 @@
+<!-- markdownlint-disable-file MD013 MD033 -->
+# Building rpm-ostree in autosd image from scratch
+
+The AutoSD is an upstream repository for Red Hat In-Vehicle OS, much like CentOS Stream is to RHEL. AutoSD is based on CentOS Stream with a few divergences. If your base system is not a CentOS, you can use a container to generate a compatible CentOS Stream 9 RPM as showed below, otherwise just skip the step and use your raw system.
+
+## Building rpm-ostree rpm on CentOS Stream9
+
+Create a **ContainerFile** with the content below.
```

Comment:
Well, our existing docs talk about using a toolbx, which is I think more ergonomic in many cases.

Reply:
/me nods

Reply:
One problem with toolbox is it doesn't support creating CentOS Stream 9 environments:

https://github.com/containers/toolbox/issues/275

It's one of the reasons I use the derivative distrobox because it can spin up virtually any distro. Or sometimes I just do podman run --rm -v $HOME:$HOME.

Of course you might be able to run mock inside toolbox to generate the cs9 rpms.

In the past for some reason I've always found podman run --rm -v $HOME:$HOME some_cs9_container and rpmbuild inside the container easiest

Reply:
But it does seem like this was fixed since I last played around with toolbox:

https://github.com/containers/toolbox/issues/1019

so toolbox sounds good.